concent_api/core/tests/test_integration_subtask_results_verify.py

```diff
@@ -0,0 +1,228 @@
+            response,
+            status=200,
+            key=self.PROVIDER_PRIVATE_KEY,
+            message_type=message.concents.ServiceRefused,
+            fields={
+                'reason': message.concents.ServiceRefused.REASON.DuplicateRequest,
+            }
+        )
+        self._assert_stored_message_counter_not_increased()
+
+    def test_that_concent_reponds_with_insufficient_funds_when_requestor_doesnt_have_funds(self):
+        # given
+        (serialized_subtask_results_verify,
+         subtask_results_verify_time_str) = self._create_serialized_subtask_results_verify()
+
+        # when
+        with mock.patch("core.message_handlers.base.is_requestor_account_status_positive", return_value=False):
+            with freeze_time(subtask_results_verify_time_str):
+                response = self.client.post(
+                    reverse('core:send'),
+                    data=serialized_subtask_results_verify,
+                    content_type='application/octet-stream',
+                    HTTP_CONCENT_CLIENT_PUBLIC_KEY=self._get_encoded_provider_public_key(),
+                    HTTP_CONCENT_OTHER_PARTY_PUBLIC_KEY=self._get_encoded_requestor_public_key(),
+                )
+
+        # then
+        self._test_response(
+            response,
+            status=200,
+            key=self.PROVIDER_PRIVATE_KEY,
+            message_type=message.concents.ServiceRefused,
+            fields={
+                'reason': message.concents.ServiceRefused.REASON.TooSmallRequestorDeposit,
+            }
+        )
+        self._assert_stored_message_counter_not_increased()
+
+    def test_that_concent_responds_with_service_refused_when_requestor_does_not_complain_about_verification(self):
+        # given
+        (serialized_subtask_results_verify,
+         subtask_results_verify_time_str) = self._create_serialized_subtask_results_verify(
+            reason_of_rejection=message.tasks.SubtaskResultsRejected.REASON.ResourcesFailure)
+
+        # when
+        with mock.patch("core.message_handlers.base.is_requestor_account_status_positive", return_value=True):
+            with freeze_time(subtask_results_verify_time_str):
+                response = self.client.post(
+                    reverse('core:send'),
+                    data=serialized_subtask_results_verify,
+                    content_type='application/octet-stream',
+                    HTTP_CONCENT_CLIENT_PUBLIC_KEY=self._get_encoded_provider_public_key(),
+                    HTTP_CONCENT_OTHER_PARTY_PUBLIC_KEY=self._get_encoded_requestor_public_key(),
+                )
+
+        # then
+        self._test_response(
+            response,
+            status=200,
+            key=self.PROVIDER_PRIVATE_KEY,
+            message_type=message.concents.ServiceRefused,
+            fields={
+                'reason': message.concents.ServiceRefused.REASON.InvalidRequest,
+            }
+        )
+        self._assert_stored_message_counter_not_increased()
+
```

Comment:
Looks good, just remember to add more tests when more checks are added in view.

Reply:
:+1: